### DISCUSSION TITLE
安装最后一步SQL报错

### DISCUSSION THREAD

**Discussion by ImJingLan** [bs-community/blessing-skin-server#542] [Q&A]
### Blessing Skin 版本

6.0.2

### PHP 版本

8.1

### 出现问题时所使用的浏览器

Firefox

### 您正在使用的 Web Server

Nginx

### 您正在使用宝塔吗？

- [X] 是

### 出现了什么问题？

安装最后一步SQL报错

```SQLSTATE[HY000]: General error: 1709 Index column size too large. The maximum column size is 767 bytes. (SQL: alter table `notifications` add index `notifications_notifiable_type_notifiable_id_index`(`notifiable_type`, `notifiable_id`)) ```

### 重现步骤

输入管理员信息后点击“开始安装”，加载片刻后报错

**Comment by CharCaius** [comment]
您使用的MySQL版本号为？

**Comment by ImJingLan** [comment]
> 您使用的MySQL版本号为？
PHP 8.1

**Comment by CharCaius** [comment]
> > 您使用的MySQL版本号为？
> > PHP 8.1

我希望您告诉我您的数据库的版本而不是PHP的版本

**Comment by OakChaser** [comment]
这是MySQL的问题吧

**Comment by Minecraft-Gtb520** [comment]
我也遇到了这个问题，使用的是MYSQL5

**Comment by TResult97** [comment]
同样的问题：

> 详细信息：SQLSTATE[42000]: Syntax error or access violation: 1071 Specified key was too long; max key length is 1000 bytes (SQL: alter table `notifications` add index `notifications_notifiable_type_notifiable_id_index`(`notifiable_type`, `notifiable_id`))

环境：

- Centos 9 (64位)
- PHP 8.1
- MySQL 5.6

搜索网络后，尝试了在配置中添加了以下项目：
```
innodb_file_format=Barracuda
innodb_file_per_table = 1
innodb_large_prefix=1
innodb_strict_mode=1
```
无法解决，还是同样的报错。
